### DISCUSSION TITLE
Capytaine added mass to WAMIT openfast input files

### DISCUSSION THREAD

**Discussion by Antonx963** [capytaine/capytaine#463] [Q&A]
Hi, 

I'm trying to convert the added mass output of capytaine to input files to be used in the open source time-domain solver for floating wind turbines, openfast. When i try to do this however, only my masses on the diagonal seem to correspons somewhat with the examples given by openfast. The results are non-dimensionalised by dividing the masses by rho = 1025 (characteristic length was 1). Any idea what I'm doing wrong? Will provide my output, code and the original WAMIT output given by openfast.

Thanks!

[marin_semi.1.txt](https://github.com/capytaine/capytaine/files/14299317/marin_semi.1.txt)
[marin_semi(original).1.txt](https://github.com/capytaine/capytaine/files/14299325/marin_semi.original.1.txt)
[deepcwind.txt](https://github.com/capytaine/capytaine/files/14299361/deepcwind.txt)

**Comment by mancellin** [comment]
Sorry for the late answer.

I noticed that the 
```python
hydrostatics1 = body1.compute_hydrostatics(rho=1025.0)
```
near line 51 is probably overwriting the matrices you defined manually above.
But that is independent from the added mass and radiation damping.

From a quick look at the files you shared, it seems that the mismatch is low for translation dofs and much higher for rotation dofs. Could you check if the same rotation center was used in both cases?

Otherwise, could you also share the mesh for me to try to run the code?
